### ISSUE TITLE
Showing Black Screen on Android

### ISSUE THREAD

**Issue by rizbud** [bortolilucas/react-native-jitsimeet#23]
The Jitsi Meet neither JitsiMeetView nor Jitsi Extended Activity just is showing black screen on android. But, working fine on ios.
```
"react-native": "0.65.1"
"@bortolilucas/react-native-jitsimeet": "2.1.1"
```

**Comment by rizbud** [comment]
hello @bortolilucas?

**Comment by jdpithwa** [comment]
Hello @bortolilucas : Could you please help here? it's urgent issue we are facing this issue in production. it would be great if you can help asap.

**Comment by bortolilucas** [comment]
@jdpithwa @rizbud  Do you get any errors in the logcat or console?

**Comment by bortolilucas** [comment]
Maybe its a version mismatch with JitsiMeetSDK

**Comment by jdpithwa** [comment]
> @jdpithwa @rizbud Do you get any errors in the logcat or console?

@bortolilucas: Nope, not getting any error log in logcat or console. basically, <JitsiMeetView> launched but this ```
onConferenceJoined={(e) => console.log(e.nativeEvent)}
        onConferenceWillJoin={(e) => console.log(e.nativeEvent)}
``` methods are not called. but when we close this <JitsiMeetView> and come again second time. it works.

**Comment by jdpithwa** [comment]
> JitsiMeetSDK

@bortolilucas : Apart from installing https://www.npmjs.com/package/@bortolilucas/react-native-jitsimeet do we need to install/add any additional JitsiMeetSDK for this?

Could you please elaborate this?

**Comment by bortolilucas** [comment]
> > JitsiMeetSDK
> 
> @bortolilucas : Apart from installing https://www.npmjs.com/package/@bortolilucas/react-native-jitsimeet do we need to install/add any additional JitsiMeetSDK for this?
> 
> I am using react-native version 66.4.
> 
> Could you please confirm this?

Apart from the steps declared in the README.md, no additional installation is required

**Comment by bortolilucas** [comment]
Have you tried both modes this package offers?

**Comment by bortolilucas** [comment]
Or any os these solutions? https://github.com/skrafft/react-native-jitsi-meet/issues/117#issuecomment-636264290

**Comment by whitmanlui** [comment]
I had the same issue, even I downgrade to RN 0.66. onConferenceJoined is called, but stills showing black screen
